src/loaders/LoadingManager.js

```diff
@@ -13,6 +13,8 @@ function LoadingManager( onLoad, onProgress, onError ) {
 	this.onProgress = onProgress;
 	this.onError = onError;
 
+	this.resourceTransform = undefined;
```

Comment:
I know you're following the pattern (sorry about that) but, any reason this needs to be public?

Reply:
Done 👍